### ISSUE TITLE
Doesn't do anything

### ISSUE THREAD

**Issue by rmeekers** [ValentinHLica/writer-gpt#1]
When I follow the installation instructions, this is all the output that the app is generating:

```
rutgermeekers@Rutgers-MacBook-Pro chatgpt_article_writer % npm start

> writer-gpt@1.0.0 start
> ts-node src/index.ts

✅ 🏗️  Building Structure for (obfuscated title)...
✅ 🏗️  Building Structure for (obfuscated title)...
✅ 🏗️  Building Structure for (obfuscated title) ...
rutgermeekers@Rutgers-MacBook-Pro chatgpt_article_writer % 
```

No files are generated in the output folder. Any idea what's going on?

**Comment by ValentinHLica** [comment]
please check if folder output exists

**Comment by rmeekers** [comment]
I have this:

<img width="184" alt="image" src="https://user-images.githubusercontent.com/5755308/233625714-374203c2-7ecf-4ff7-9c34-003dcae21a43.png">

.env file is also present.

**Comment by ValentinHLica** [comment]
> I have this:
> 
> <img alt="image" width="184" src="https://user-images.githubusercontent.com/5755308/233625714-374203c2-7ecf-4ff7-9c34-003dcae21a43.png">
> 
> .env file is also present.

in this image there is not .env file, please check if you can view hidden files in macos file explorer

**Comment by fahimanwer** [comment]
> I have this:
> 
> <img alt="image" width="184" src="https://user-images.githubusercontent.com/5755308/233625714-374203c2-7ecf-4ff7-9c34-003dcae21a43.png">
> 
> .env file is also present.

Goto the folder and press 'CMD+SHIFT+.' it will show all the hidden files.

**Comment by rmeekers** [comment]
<img width="202" alt="image" src="https://user-images.githubusercontent.com/5755308/233630706-a97cf0a1-223c-4ba5-b25b-800ab1de4542.png">

**Comment by rmeekers** [comment]
with the new version I have this error:

`The model: `gpt-4` does not exist`

**Comment by fahimanwer** [comment]
> with the new version I have this error:
> 
> ```
> The model: `gpt-4` does not exist
> ```

The issue is with your OpenAI account this tool isuing GPT-4 API request and your account is not yet approved for GPT-4 API.

**Comment by rmeekers** [comment]
I've updated it to 3.5, but it stops (hangs) during the generation of an article:

<img width="205" alt="image" src="https://user-images.githubusercontent.com/5755308/233634109-edce7854-4373-421b-b771-8a9c0d788c70.png">

**Comment by fahimanwer** [comment]
Wait it takes around 10min to generate an article.

**Comment by rmeekers** [comment]
I had listed 3 articles, but none of them are complete. They all end in the middle of a sentence

**Comment by fahimanwer** [comment]
Yeah, that is a known issue with 3.5 I am also facing it and it is not only in this tool but also in some others and only with GPT3.5 it stops after introduction.

**Comment by ValentinHLica** [comment]
> I had listed 3 articles, but none of them are complete. They all end in the middle of a sentence

Ill try to implement gtp3.5 aswell
